make.bat

```diff
@@ -156,4 +157,18 @@ EXIT /B %ERRORLEVEL%
 
 :check
 call :lint && call :test
-EXIT /B %ERRORLEVEL%
\ No newline at end of file
+EXIT /B %ERRORLEVEL%
+
+
+:docstrings
+sphinx-apidoc --ext-autodoc ^
+ --ext-intersphinx ^
+ --ext-mathjax ^
+ --extension sphinx.ext.napoleon ^
+ --extension matplotlib.sphinxext.plot_directive ^
+ --full --separate --module-first ^
+ -o test_docs %PACKAGE_DIR% && ^
+ sphinx-build -nWT test_docs\ test_docs\_build\
```

Comment:
```suggestion
sphinx-build -nWT test_docs\ test_docs\_build\
```

This is amazing! Small nitpick otherwise it gives error ``' sphinx-build' is not recognized...``.

I was trying to build docs on my windows system and got the error

```none
Warning, treated as error:
autodoc: failed to import module 'pymc4'; the following exception was raised:
No module named 'pymc4'
```

and I am not able to build docs. Is there a way to get around this?

Reply:
Thanks @tirthasheshpatel. It looks like you hadn't installed pymc4 before trying to build the docs. You need to install it first

Reply:
Thanks! Works now!